lib/spack/spack/cmd/test.py

```diff
@@ -8,8 +8,11 @@
 import argparse
 import textwrap
 import datetime
+import fnmatch
+import re
 
 import llnl.util.tty as tty
+from llnl.util.filesystem import mkdirp
```

Comment:
I thought we weren't supposed to be doing `from import` statements anymore.

Reply:
We had a discussion about limiting them, but this seems to be our idiom for this particular utility.

```
$ grep -rn "import mkdirp" lib/spack/spack/ | wc -l
      38
```

Reply:
I'm getting rid of it anyway as I refactor.

Reply:
Okay.